.github/CODEOWNERS

```diff
@@ -33,6 +33,7 @@ ddtrace/contrib/pytest_bdd          @DataDog/ci-app-libraries
 ddtrace/contrib/unittest            @DataDog/ci-app-libraries
 tests/contrib/pytest                @DataDog/ci-app-libraries
 tests/contrib/pytest_bdd            @DataDog/ci-app-libraries
+tests/contrib/grpc                  @DataDog/ci-app-libraries @DataDog/asm-python
```

Comment:
Why is this a @DataDog/ci-app-libraries thing and if it is, why wasn't a reviewer from that team added?